src/java_tools/buildjar/java/com/google/devtools/build/buildjar/SimpleJavaLibraryBuilder.java

```diff
@@ -211,6 +211,7 @@ public FileVisitResult visitFile(Path path, BasicFileAttributes attrs)
     if (protobufMetadataBuffer.size() > 0) {
       try (OutputStream outputStream = Files.newOutputStream(output)) {
         protobufMetadataBuffer.writeTo(outputStream);
+        outputStream.close();
```

Comment:
The try-with-resources should close the output stream, why is this necessary?

Reply:
Oh, sorry I didn't mean to make this change. I was just debugging the Java build failure, but yes, looks like this should not be necessary.